tests/Makefile

```diff
@@ -10,7 +10,13 @@ FAKE_GCS_SERVER_ROOT := /tmp/fake-gcs-server
 FAKE_GCS_SERVER_DATA_DIR := ${FAKE_GCS_SERVER_ROOT}/data
 FAKE_GCS_SERVER_BUCKET := test-bucket
 
-.PHONY: gcs s3 fuzzing all
+AZURITE_API_PORT := 10000
+AZURITE_ACCOUNT_NAME := devstoreaccount1
+# Well known storage account key https://github.com/Azure/Azurite?tab=readme-ov-file#default-storage-account
+AZURITE_ACCOUNT_KEY := Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+AZURITE_BUCKET := test-bucket
```

Comment:
I suggest calling it `AZURITE_CONTAINER` to be aligned with what it really called, also in the `runai_model_streamer_azure` package